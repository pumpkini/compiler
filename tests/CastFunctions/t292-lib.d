int main(){
    Print(btoi(true));
    Print(btoi(false));
    Print(btoi(!true));
    Print(itob(2));
    Print(itob(0));
    Print(itob(1-1));
    Print(dtoi(2.5));
    Print(dtoi(2.1));
    Print(dtoi(2.6));
    Print(dtoi(2.3)*3);
    Print(itod(5));
    Print(itod(10));
    Print(itod(5)/2.5);
    Print(itod(5)/4.0);
    Print( dtoi(itod(btoi(true)) + 2.3) );
}
