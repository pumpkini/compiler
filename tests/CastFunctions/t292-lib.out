1
0
0
true
false
false
3
2
3
6
5.00000000
10.00000000
2.00000000
1.25000000
3
